task2.js

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import path from 'path';
+import csv from 'csvtojson';
+import { pipeline, Transform } from 'stream';
+
+const writeCsvToTxtLineByLine = async (fromPath, toPath) => {
+    try {
+        const transform = new Transform({
+            transform: function(chunk, encoding, callback) {
+                callback(null, converter(JSON.parse(chunk.toString())));
```

Comment:
There must be a simpler way to do the necessary transformation :)
Btw, the format is still doesn't align with
> Use the following format: https://epa.ms/nodejs19-hw1-ex2

The key must be in lower case and `amount` field should be absent

Reply:
Fixed :)